<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASGraph</class>
 <widget class="QDialog" name="ASGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASGraph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/terminal16.png</normaloff>:/terminal16.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="100,8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCustomPlot" name="qcGraph" native="true"/>
   </item>
   <item>
    <widget class="QFrame" name="frData">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbItemCount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max data point:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbItemCount">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>7</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbResetZoom">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>6</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom Reset</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbLineStyle">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>6</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Line Style:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbLineStyles">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>6</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Line</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>StepLeft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>StepRight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>StepCenter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Impulse</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pbSet">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>6</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbClear">
      <property name="geometry">
       <rect>
        <x>539</x>
        <y>6</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbSetXY">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>7</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbXFrom">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>6</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-10000000</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbX">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>6</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbXd">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>6</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbXTo">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>6</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-10000000</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbYFrom">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>6</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-10000000</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbYd">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>6</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbY">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>6</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbYTo">
      <property name="geometry">
       <rect>
        <x>867</x>
        <y>6</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-10000000</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPause">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>6</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
